cloudwatch.tf

```diff
@@ -87,12 +87,12 @@ data "template_file" "s3_widgets" {
 EOF
 
   vars = {
-    bucket_name = "${local.apiary_data_buckets[count.index]}"
+    bucket_name = "${local.schemas_info[count.index]["data_bucket"]}"
   }
 }
 
 data "template_file" "ecs_widgets" {
-  count = "${var.hms_instance_type == "ecs" ? length(local.apiary_data_buckets) : 0}"
+  count = "${var.hms_instance_type == "ecs" ? length(local.schemas_info) : 0}"
```

Comment:
@rpoluri - why is the `ecs_widgets` template created once per Apiary schema?   Seems like it should only be created once?

Reply:
yes, should be create only once.